manticore/ethereum/manticore.py

```diff
@@ -1625,17 +1590,8 @@ def _emit_trace_file(filestream, trace):
             ln = "0x{:x}:0x{:x} {}\n".format(contract, pc, "*" if at_init else "")
             filestream.write(ln)
 
-    @property
-    def global_findings(self):
-        global_findings = set()
-        for detector in self.detectors.values():
-            for address, pc, finding, at_init in detector.global_findings:
-                if (address, pc, finding, at_init) not in global_findings:
-                    global_findings.add((address, pc, finding, at_init))
-        return global_findings
-
     @ManticoreBase.at_not_running
-    def finalize(self, procs=10):
+    def finalize(self, procs=45):
```

Comment:
Should we pull the `procs` argument from `config` instead of hardcoding it? 45 may cause some trouble on most machines.